--- a/src/conf.c
+++ b/src/conf.c
@@ -127,9 +127,9 @@
 	}
 #else
 #ifdef __APPLE__
-	base_config_dir = strdup("/var/db");
+	base_config_dir = strdup("@TERMUX_PREFIX@/var/db");
 #else
-	base_config_dir = strdup("/var/lib");
+	base_config_dir = strdup("@TERMUX_PREFIX@/var/lib");
 #endif
 #endif
 	__config_dir = string_concat(base_config_dir, DIR_SEP_S, CONFIG_DIR, NULL);
@@ -431,7 +431,7 @@
 	remove(device_record_file);
 
 	/* store file */
-	if (!plist_write_to_file(plist, device_record_file, PLIST_FORMAT_XML, 0)) {
+	if (plist_write_to_file(plist, device_record_file, PLIST_FORMAT_XML, 0) != PLIST_ERR_SUCCESS) {
 		usbmuxd_log(LL_DEBUG, "Could not open '%s' for writing: %s", device_record_file, strerror(errno));
 		res = -ENOENT;
 	}
